docs/assistant/security-assistant.asciidoc

```diff
@@ -72,29 +70,60 @@ NOTE: All chat history and custom quick prompts persist in local browser storage
 
 [discrete]
 [[interact-with-assistant]]
-== Interact with the Security Assistant
+== Interact with the AI Assistant
 
-Use these features to adjust and act on your conversations with the Security Assistant:
+Use these features to adjust and act on your conversations with the AI Assistant:
 
-* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the Security Assistant's answers to be.
+* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the AI Assistant's answers to be.
 +
 [role="screenshot"]
 image::images/system-prompt.gif[The system prompt drop-down menu,90%]
 +
-NOTE: The system prompt is only configurable at the start of a conversation. To reconfigure it, clear the chat and start a new conversation.
+System prompts provide context to the model, informing its response. To create a custom system prompt, open the system prompts dropdown menu and click *+ Add new system prompt...*.
 
-* Select a _quick prompt_ at the bottom of the chat window to get help writing a prompt for a specific purpose, such as summarizing an alert or converting a query from a legacy SIEM to {elastic-sec}. Available quick prompts vary based on context. You can also add custom quick prompts for questions you frequently ask the Security Assistant.
+* Select a _quick prompt_ at the bottom of the chat window to get help writing a prompt for a specific purpose, such as summarizing an alert or converting a query from a legacy SIEM to {elastic-sec}.
 +
 [role="screenshot"]
 image::images/quick-prompts.png[Quick prompts highlighted below a conversation,90%]
++
+Quick prompt availability varies based on context — for example, the Alert summarization quick prompt appears when you open the AI Assistant while viewing an alert. To customize existing quick prompts and create new ones, click *Add Quick prompt*.
```

Comment:
Suggesting this styling since we bold interactive UI elements.
 
```suggestion
Quick prompt availability varies based on context — for example, the **Alert summarization** quick prompt appears when you open the AI Assistant while viewing an alert. To customize existing quick prompts and create new ones, click *Add Quick prompt*.
```